### PR TITLE
ENG-372 fix active default language overriding

### PR DESCRIPTION
**Pull request by ichalagashvili**

### CODE REVIEW

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: src/state/languages/actions.js
Comment (jeffgo10) [inline src/state/languages/actions.js:45]: why not use the object spread operator (e.g. `{ ...  getLanguagesMap(getState())[langCode] }`) instead of using `Object.assign`?

**Review by jeffgo10 — COMMENTED** [review]
only 1 clarification. Also, can you please add the new i18n items in `src/locles/pt.js` def?

File: src/state/languages/actions.js
Comment (ichalagashvili) [reply src/state/languages/actions.js:45]: its same and Assign is more expressive the purpose, but agree, spread operator is nicer. Changed.

Comment (jeffgo10) [reply src/state/languages/actions.js]: yup its actually the same. but most people here prefer es6 syntax.

**Review by jeffgo10 — APPROVED** [review]
all good

**Comment by entando-jx** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/entando/app-builder/pull/681#pullrequestreview-392564980" title="Approved">jeffgo10</a>*
To fully approve this pull request, please assign additional approvers.
We suggest the following additional approver: **ichalagashvili**

If they are not already assigned, you can assign the PR to them by writing `/assign @ichalagashvili` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=entando%2Fapp-builder).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/entando/app-builder/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
